inference-engine/src/vpu/graph_transformer/src/stages/interp.cpp

```diff
@@ -45,8 +45,12 @@ class InterpStage final : public StageNode {
 
     void serializeParamsImpl(BlobSerializer& serializer) const override {
         auto align_corners = attrs().get<bool>("align_corners");
+        auto sampleType = attrs().get<InterpolateMode>("mode");
+        auto coordinateTransMode = attrs().get<InterpolateCoordTransMode>("coordinate_transformation_mode");
```

Comment:
Nitpick: I'd suggest to use string constants for attributes names instead of literals. It makes code a bit safer since you exclude possibility of typo issues which are runtime, not compile-time in comparison with typical C++ cases

Reply:
okay. Thanks

Reply:
done